requirements.yml

```diff
@@ -190,9 +190,9 @@
   setup_options: '--offline'
   include_extras: false
 
-#- name: glueviz
-#  version: '0.9.1'
-#  python: '<3.6*'
+- name: glueviz
+  version: '0.10.0'
+  python: '<3.6*'
```

Comment:
Are you getting the build from conda-forge or building your own here? 0.10.0 should work with Python 3.6